conf/machine/qemuriscv64.conf

```diff
@@ -5,6 +5,9 @@
 PREFERRED_PROVIDER_virtual/kernel ?= "linux-riscv"
 PREFERRED_VERSION_linux-riscv ?= "4.18%"
 
+PREFERRED_PROVIDER_virtual/bootloader ?= "u-boot"
+PREFERRED_PROVIDER_u-boot ?= "linux-riscv"
```

Comment:
hello @kraj
I think PREFERRED_PROVIDER_u-boot parament value is "u-boot" (line:9)
Can you check it again please.